api/api/logic/suggestions.py

```diff
@@ -101,14 +100,15 @@ def post_suggestion() -> str:
     response = created_response[0]
 
     if response is not None and response['code'] is 201:
-      suggestion_id = response['data']['id']
-      protocol = connexion.request.environ['HTTP_X_FORWARDED_PROTO']
-      baseurl = connexion.request.environ['HTTP_HOST'].split(',')[1]
+        suggestion_id = response['data']['id']
+        protocol = connexion.request.environ['HTTP_X_FORWARDED_PROTO']
+        baseurl = connexion.request.environ['HTTP_HOST'].split(',')[1]
+
+    if suggestion_id > 0 and protocol is not '' and baseurl is not None and baseurl is not '':
+        response['data']['suggestionUrl'] = f'{protocol}://{baseurl}/suggestion/{suggestion_id}'
 
-      if suggestion_id > 0 and protocol is not '' and baseurl is not None and baseurl is not '':
-        response['suggestionUrl'] = f'{protocol}://{baseurl}/suggestions/{suggestion_id}'
+    return jsonify(response['data']), 201
```

Comment:
voi olla että katon väärin identtejä mutta pitäiskö tän palauttaa jos create response palauttaa 404 niin se ulospäin, nythän tää taitaa createn yhteydessäkin palauttaa 201?

Reply:
korjattu

Reply:
korjattu